wallets/rn_cli_wallet/src/components/SubscriptionList.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import {RefreshControl, ScrollView} from 'react-native';
+
+import {useSnapshot} from 'valtio';
+import {useNavigation} from '@react-navigation/native';
+
+import useNotifyClientContext from '@/hooks/useNotifyClientContext';
+import SubscriptionItem from '@/components/components/SubscriptionItem';
+import SettingsStore from '@/store/SettingsStore';
+
+export default function SubscriptionList() {
+  const {subscriptions, fetchSubscriptions} = useNotifyClientContext();
+  const [refreshing, setRefreshing] = React.useState(false);
+  const {eip155Address: address} = useSnapshot(SettingsStore.state);
+  const {navigate} = useNavigation();
+
+  async function handleRefresh() {
+    setRefreshing(true);
+    await fetchSubscriptions();
+    setRefreshing(false);
+  }
+
+  return (
+    <ScrollView
```

Comment:
same here, this is a good case to use Flatlist.

ScrollViews are typically used for views that contain a lot of different components, and FlatLists to render arrays of similar components

Reply:
Exactly